### PR TITLE
README.md: Update language around PoC

### PR DESCRIPTION
**Pull request by mestery**
This commit updates the README.md file to reflect the reality of the
initial PoC work being done using containers and IPDK P4-eBPF.

Signed-off-by: Kyle Mestery <mestery@mestery.com>

### CODE REVIEW

File: sw_firewall/README.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+
+- traffic sources
+- traffic targets
+- proxies
+
+The initial PoC will be:
+
+- IPDK P4-eBPF container for networking
+- Docker-based container setup using Docker Compose
+- Server containers hostring traffic sources
```
Comment (seroyer) [inline sw_firewall/README.md:13]: s/hostring/hosting/

**Review by seroyer — COMMENTED** [review]
Minor typo, otherwise looks good.

**Review by seroyer — APPROVED** [review]

File: sw_firewall/README.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
 
-The Proof of Concept is made up of three types of containers: traffic sources,
-traffic targets, and proxies.  The initial PoC will be:
+The Proof of Concept is made up of three types of containers:
+
+- traffic sources
+- traffic targets
+- proxies
+
+The initial PoC will be:
```
Comment (mkalderon) [inline sw_firewall/README.md:9]: Perhaps Initial PoC should also be marked as title similar to Second PoC with "##"

File: sw_firewall/README.md [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+- Traffic target containers
+- Proxy containers
+
+## Second PoC
+
+The Second Proof of Concept is made up of three types of containers: traffic
+sources, traffic targets, and proxies.  This second PoC will be:
 - Servers hosting traffic sources sending traffic to
 - Servers with IPU or DPU cards, with traffic targets on the servers
 - Proxies running on IPU or DPU cards
```
Comment (mkalderon) [inline sw_firewall/README.md:24]: Is this really a second PoC? Or is it a PoC on top of the initial p4-ebpf one ? 
Don't both of these together provide the FireWall PoC?

**Review by mkalderon — COMMENTED** [review]
sorry for the late review - some questions inside

File: sw_firewall/README.md (hunk 3 above)
Comment (mestery) [reply sw_firewall/README.md:24]: The main separation is that they are in fact two different PoCs, and we may not use p4-ebpf on top of IPU/DPU for the second PoC.

Comment (mkalderon) [reply sw_firewall/README.md:24]: It's not clear in the flow of the README, there is a description of first poc and second poc, then running the ipdk PoC
Perhaps moving the "Running the PoC" Before elaborating on the second PoC will clarify that ? 
Or separate files per PoC ?

Comment (mkalderon) [reply sw_firewall/README.md:24]: sure

**Pull request merged**